src/ui/public/courier/fetch/request/__tests__/segmented.js

```diff
@@ -0,0 +1,55 @@
+describe('ui/courier/fetch/request/segmented', () => {
+  const sinon = require('auto-release-sinon');
+  const expect = require('expect.js');
+  const ngMock = require('ngMock');
+
+  let _Promise;
+  let $rootScope;
+  let SegmentedReq;
+  let segmentedReq;
+  let searchReqStart;
+
+  beforeEach(ngMock.module('kibana'));
+
+  beforeEach(ngMock.inject((Private, Promise, $injector) => {
+    _Promise = Promise;
```

Comment:
Mind doing the same thing you did for `$rootScope` here? Using `_Promise` all over is sort of strange.